src/bin/ion/commands/beta/to/json.rs

```diff
@@ -57,75 +41,44 @@ pub fn run(_command_name: &str, matches: &ArgMatches) -> Result<()> {
                 output_file
             )
         })?;
-        Box::new(file)
+        Box::new(BufWriter::new(file))
```

Comment:
🗺️ Previously the `to json` command was not using buffered output except when writing to `STDOUT`.